tests/storage/mpathconf_test.py

```diff
@@ -102,3 +102,64 @@ def test_read_bad_blacklist(fake_conf):
     fake_conf.write(BAD_CONF)
     wwids = mpathconf.read_blacklist()
     assert wwids == {"wwid1", "wwid2", "wwid5"}
+
+
+@pytest.fixture
+def fake_mpath_conf(tmpdir, monkeypatch):
+    fake_conf = tmpdir.join("multipath.conf")
+    monkeypatch.setattr(mpathconf, "CONF_FILE", str(fake_conf))
+    return fake_conf
+
+
+def test_mpath_current_tag_conf(fake_mpath_conf):
+    mpath_conf_current_tag = """\
+# VDSM REVISION 2.2
+"""
+    fake_mpath_conf.write(mpath_conf_current_tag)
+    assert mpathconf.read_metadata() == mpathconf.Metadata(
+        revision=mpathconf.REVISION_OK,
+        private=False,
+    )
+
+
+def test_mpath_private_current_tag_conf(fake_mpath_conf):
+    mpath_conf_current_private = """\
```

Comment:
When the variable is the context of a single test we don't need the long name, "data" or "text" is better.

The name of the tests explains the data or text used here.